roles/user/tasks/main.yml

```diff
@@ -4,10 +4,16 @@
   file:
     path: "{{ global_base_path }}"
     state: directory
+    owner: root
+    group: root
+    mode: 0755
 - name: Create local applications directory
   file:
     path: /usr/local/share/applications/
     state: directory
+    owner: root
+    group: root
```

Comment:
This seems to have been group "staff" before, but everything else in that hierarchy is owned by "root". Just noting the change, but I think it's better this way.

Reply:
With `0755` there also isn't a difference I'm aware of for a typical user between its group being `staff` vs `root`, even when a member of `staff`. On my Debian box, the only thing that isn't `0755` and `root:root` in `/usr/local/share` is `fonts/` which is `2775` and `root:staff`.

On a fresh Linux Mint, `/usr/local/share/applications/` doesn't seem to exist, so `0755` and `root:root` feels like the best option until we're proven otherwise.